features/hydra/collection.feature

```diff
@@ -273,6 +273,44 @@ Feature: Collections support
     }
     """
 
+  @!mongodb
```

Comment:
Why not MongoDB too?

Reply:
because it's testing the metadata which doesn't impact mongodb or doctrine in particular. + it reduces time to test